packages/gasket-plugin-dynamic-plugins/lib/index.js

```diff
@@ -0,0 +1,36 @@
+import { createRequire } from 'module';
+const require = createRequire(import.meta.url);
+const { name, version } = require('../package.json');
+
+/** @type {import('@gasket/core').Plugin} */
+export default {
+  name,
+  version,
+  hooks: {
+    async prepare(gasket) {
+      if (
+        !process.env.GASKET_DYNAMIC
+        || !gasket.config.dynamicPlugins
+        || !gasket.config.dynamicPlugins.length
```

Comment:
See other comment, but I think we should base execution on the resulting config-only and use the `GASKET_ENV` to help manage that.

This check could then be simplified to
```suggestion
        !gasket.config.dynamicPlugins?.length
```